Dockerfile

```
@@ -1,11 +1,11 @@
-FROM node:16-buster-slim
+FROM node:18-bullseye-slim
```

We recommend being as specific as possible with Docker images, to reduce the chance of a new deploy accidentally breaking (we've seen this happen multiple times).

```suggestion
FROM node:18.9.1-bullseye-slim
```

---

Done.